src/components/example/NewExampleForm.vue

```diff
@@ -311,32 +303,55 @@ export default {
 </script>
 
 <style lang="scss" scoped>
+@import "~@weni/unnnic-system/src/assets/scss/unnnic.scss";
+.fields {
+  &__container {
+    margin-block: $unnnic-spacing-md;
+  }
 
-.language-append {
-  flex-grow: 0;
-}
+  &__header {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: $unnnic-spacing-md;
+    gap: $unnnic-spacing-sm;
 
-.wrapper {
-  margin: 1rem 0;
-}
+    > * {
+      flex: 1;
+      min-width: 15.3125 * $unnnic-font-size;
+    }
 
-.columns.is-variable .column {
-  padding-left: 0;
-  padding-right: 0;
-}
+    > *:first-child {
+      flex: 100;
+      min-width: 31.6875 * $unnnic-font-size;
```

Comment:
Why "31.6875"? try adding this value into a variable and name it to describe this value